docs/developing_iree/profiling_cpu_events.md

```diff
@@ -0,0 +1,13 @@
+# CPU cache and other CPU event profiling
```

Comment:
How is this going to appear in the sidebar on our GitHub Pages site?

![image](https://user-images.githubusercontent.com/4010439/101057202-a48fd200-3540-11eb-9e1b-bdbad6d9b33f.png)

The profiling sections should be grouped, either by explicit ordering: https://github.com/google/iree/blob/dc84fff402c6dc98c1406ba6b0be0c5a2a33fb5e/scripts/prepare_doc_publication.py#L113-L120

or by having similar titles.

One small change that could help would be starting each title with "Profiling":
* Profiling
* Profiling CPU caches and events
* Profiling Vulkan GPUs

This is where titles can be set differently from the markdown `# title`
https://github.com/google/iree/blob/dc84fff402c6dc98c1406ba6b0be0c5a2a33fb5e/scripts/prepare_doc_publication.py#L44-L65

Reply:
Thanks for bringing this up, I didn't realize at all.